plugin/src/main/java/io/jenkins/plugins/casc/impl/configurators/HeteroDescribableConfigurator.java

```diff
@@ -117,6 +117,18 @@ public CNode describe(T instance, ConfigurationContext context) {
                 }).getOrNull();
     }
 
+    @CheckForNull
+    public void describeStructure(T instance, ConfigurationContext context) {
+        lookupConfigurator(context, instance.getClass())
+            .map(configurator -> convertToNode(context, configurator, instance))
+            .filter(Objects::nonNull)
+            .map(node -> {
+                    return new Scalar(preferredSymbol(instance.getDescriptor()));
+            }).getOrNull();
+
+        System.out.println(instance.getDescriptor());
```

Comment:
Should this be used in a different context. I could not quite use the `getDescriptorList`.